bin/migrate-tips.py

```diff
@@ -3,14 +3,20 @@
 
 db = db(env())
 
-slugs = db.all("""
-    SELECT slug
-      FROM teams
-     WHERE is_approved IS TRUE
+teams = db.all("""
+    SELECT distinct ON (t.slug) t.*::teams
+      FROM teams t
+      JOIN tips ON t.owner = tips.tippee	-- Only fetch teams whose owner have tips.
+     WHERE t.is_approved IS TRUE           	-- Only fetch approved teams.
+       AND NOT EXISTS (                  	-- Make sure not already migrated.
+            SELECT 1
```

Comment:
@rohitpaulk do you have any idea why the comment looks funny here but looks fine in the editor?
![screenshot from 2016-01-24 15 25 09](https://cloud.githubusercontent.com/assets/4983347/12535604/b174e49c-c2ae-11e5-975c-e08107be30de.png)